common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -709,3 +713,72 @@ def associate_by_email_if_login_api(auth_entry, backend, details, user, current_
             # email address and the legitimate user would now login to the illegitimate
             # account.
             return association_response
+
+
+def user_details_force_sync(auth_entry, strategy, details, user=None, *args, **kwargs):
+    """
+    Update normally protected user details using data from provider.
+
+    This step in the pipeline is akin to `social_core.pipeline.user.user_details`, which updates
+    the user details but has an unconfigurable protection over updating the username & email, and
+    is unable to update information such as the user's full name which isn't on the user model, but
+    rather on the user profile model.
+
+    Additionally, because the email field is normally used to login, if the email is changed by this
```

Comment:
@UmanShahzad Minor nit: "login" should be "log in" (since it's used as a verb here).

*Disclaimer*: I'm generally more of a [descriptivist than a prescriptivist](http://english.blogoverflow.com/2012/10/prescriptivism-and-descriptivism/) but misspellings of "setup" (noun)/"set up" (verb) and "login"/"log in" tend to make me cringe for some reason :)

Reply:
Ah, sure! :+1: